src/System.Collections/tests/Resources/System.Collections.Tests.rd.xml

```diff
@@ -1,7 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
-  <Library>
+  <Library Name="System.Collections.Tests">
+    <Assembly Name="*Application*" DoNotInline="true" DoNotOptimize="true" />
```

Comment:
> Oh, okay. The title of bug 472947 says it's for System.Data.SqlClient.Tests and this is System.Collections (plus that bug is only 3 days old, while this workaround has been here for a while) - that's why the confusion.

Yup, sorry about that, I didn't open the bug they did in the codegen team and I was following up on email getting a consistent repro with them.